lang/types/value.go

```diff
@@ -212,7 +223,7 @@ func ValueOf(v reflect.Value) (Value, error) {
 		}, nil
 
 	default:
-		return nil, fmt.Errorf("unable to represent value of %+v", v)
```

Comment:
what? how did this ever compile with `v` before?

Reply:
I renamed the first function arg because we were pointlessly declaring it.